opennms-web-api/src/main/java/org/opennms/web/svclayer/support/DefaultNodeListService.java

```diff
@@ -160,6 +162,13 @@ private void addCriteriaForCommand(OnmsCriteria criteria, NodeListCommand comman
         }
     }
 
+    private void addCriteriaForFlows(final OnmsCriteria criteria, final Boolean hasFlows) {
+        if (hasFlows == null) {
```

Comment:
This is a duplicated check, as this is only invoked after `command.hasFlows()` is true, so not required to invoke it again. If you absolutely want this check, I'd opt for 

```java
if (hasFlows != null) {
 ...
}
```
as it is much easier to read.